### PR TITLE
[#13] [UI] As a user, I can see the iOS 11 UIKit Screen with Dynamic Custom Font

### LINKED ISSUE #13: [UI] As a user, I can see the iOS 11 UIKit Screen with Dynamic Custom Font
## Why

To display dynamic custom font with iOS 11 and UIKit.

## Acceptance Criteria

Show a screen with:
- Image view
- Title "iOS 11".
- Bold title text "Font Name"
- Text "iOS Version"
- Text "UIKit"
- Caption text "A demo from NimbleHQ"

Screen should be in vertical scroll view to support multiple screen sizes.

## Design

iOS11

### PR DESCRIPTION
**Pull request by blyscuit**
close #13 

## What happened

Show a screen with:
- Image view
- Title "iOS 11".
- Bold title text "Font Name"
- Text "iOS Version"
- Text "UIKit"
- Caption text "A demo from NimbleHQ"
 
## Insight

Use R.swift for image.

### CODE REVIEW

File: CustomDynamicFont/Sources/Presentation/Modules/iOS11UIKit/IOS11UIKitViewController.swift [hunk 1]
```diff
@@ -8,4 +8,87 @@
...
+        versionLabel.font = .customFont(UIFont.ZenOldMincho.regular, forTextStyle: .body)
+        lifecycleLabel.font = .customFont(UIFont.ZenOldMincho.regular, forTextStyle: .body)
+        commentLabel.font = .customFont(UIFont.ZenOldMincho.regular, forTextStyle: .footnote)
+
+        fontNameLabel.text = "Font Name" // TODO: Localize in integration
+        versionLabel.text = R.string.localizable.ios11uikitVersionlabelTitle()
+        lifecycleLabel.text = R.string.localizable.ios11uikitLifecyclelabelTitle()
+        commentLabel.text = R.string.localizable.ios11uikitCommentlabelTitle()
+
+        imageView.image = R.image.color_Rectangle()
```
Comment (minhnimble) [inline CustomDynamicFont/Sources/Presentation/Modules/iOS11UIKit/IOS11UIKitViewController.swift:92]: Just curious, we are going to have only one image `color_Rectangle` for this `imageView`, is that correct? If not and we need to change it later on in integrate task then we might need a TODO here as well 🙏

**Review by minhnimble — APPROVED** [review]
The rest LGTM 🚀

File: CustomDynamicFont/Sources/Presentation/Modules/iOS11UIKit/IOS11UIKitViewController.swift (hunk 1 above)
Comment (blyscuit) [reply CustomDynamicFont/Sources/Presentation/Modules/iOS11UIKit/IOS11UIKitViewController.swift:92]: Yes we will have only one image view. The image view is to show the texts' layout and the image only serves as aesthetic purpose.

**Review by markgravity — APPROVED** [review]
LGTM 👍

**Pull request merged**